packages/playwright/src/matchers/toMatchAriaSnapshot.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright Microsoft Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import type { LocatorEx } from './matchers';
+import type { ExpectMatcherState } from '../../types/test';
+import type { MatcherResult } from './matcherHint';
+import type { AriaTemplateNode, AriaTemplateString } from 'playwright-core/lib/server/injected/ariaSnapshot';
```

Comment:
```suggestion
import type { AriaTemplateNode, AriaTemplateString } from 'playwright-core/src/server/injected/ariaSnapshot';
```

Reply:
I think we are using lib when importing types.